src/api.py

```
@@ -6,9 +6,12 @@
 import os
 import json
 import shutil
+import tempfile
 from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)